src/app/(root)/favorite/setting/page.tsx

```diff
@@ -0,0 +1,29 @@
+import HeaderArrowToGoBack from "@/src/components/header-arrow-to-go-back";
+import SettingList from "@/src/components/setting-list";
+
+const page = () => {
+  return (
+    <>
+      <div className="relative h-16">
+        <HeaderArrowToGoBack path={"favorite/"} color="text-black" text="設定" border={true} />
+      </div>
```

Comment:
TopBarという共通コンポーネント用意しているので、そちらで書き換えてほしいです！

```suggestion
      <TopBar
        leadingComponent={
          <div className="flex items-center gap-3">
            <Link href={"/favorite"}>
              <ArrowLeft size={20} />
            </Link>
            <h1 className="font-bold text-mauve12 md:text-xl">設定</h1>
          </div>
        }
      />

```